monitoring/src/check-page/common-functions.ts

```diff
@@ -360,7 +363,7 @@ export const sendMetricData = async (
 	log_info(`config.platform.toUpperCase() ${config.platform.toUpperCase()})`);
 	const region = config.region;
 	const client = new CloudWatchClient({ region: region });
-	const params = {
+	const params: PutMetricDataCommandInput = {
```

Comment:
You can also use [the `satisfies` operator](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-4-9.html#the-satisfies-operator) here if you want a narrower type.

Reply:
Thanks 👍